config/features.yml

```diff
@@ -19,6 +19,10 @@ features:
     actor_type: user
     description: Use lighthouse instead of EVSS to view/download benefit letters.
     enable_in_development: true
+  benefits_documents_use_lighthouse:
```

Comment:
This is a nit, but should we consider standardizing our feature flag names to include something like `benefits_management_*` so we can easily find them?